Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -327,12 +369,12 @@ void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* m, float
 
     if (pathObj->_count + 1 >= pathObj->_max) {
```

Comment:
Yes.  That would work. The reason i suggested the internal structure is that it would then make size calculation etc trivial. 

---
In reply to: [68820359](https://github.com/Microsoft/WinObjC/pull/559#discussion_r68820359) [](ancestors = 68820359)